src/Compilers/CSharp/Test/Emit/Emit/DeterministicTests.cs

```diff
@@ -166,11 +165,13 @@ public void CompareAllBytesEmitted_Release()
             {
                 var result1 = EmitDeterministic(CompareAllBytesEmitted_Source, Platform.AnyCpu32BitPreferred, pdbFormat, optimize: true);
                 var result2 = EmitDeterministic(CompareAllBytesEmitted_Source, Platform.AnyCpu32BitPreferred, pdbFormat, optimize: true);
-                AssertEx.Equal(result1, result2);
+                AssertEx.Equal(result1.pe, result2.pe);
+                AssertEx.Equal(result1.pdb, result2.pdb);
```

Comment:
We could add overloads for AssertEx.Equal on a tuples (few of low cardinality).

Reply:
Perhaps in future, if we find more cases where we need it. Right now having more overloads doesn't seem that important.

---
In reply to: [137343046](https://github.com/dotnet/roslyn/pull/21712#discussion_r137343046) [](ancestors = 137343046)